tools/buckconfigs/windows-x86_64/type/release.bcfg

```diff
@@ -1,17 +1,17 @@
-[project]
-  buck_out = "buck-out/release"
-
-[cxx_type]
-  cppflags = \
-    /DNDEBUG
-
-  cflags = \
-    /Ot
-
-  cxxppflags = \
-    /DNDEBUG
-
-  cxxflags = \
-    /Ot
-
-  ldflags =
+[project]
+  buck_out = buck-out/release
+
+[cxx_type]
```

Comment:
Any idea what changed here? Is it whitespace differences.

Reply:
It looks like this file was created with CRNLs, when I edited they were removed.